.gitignore

```diff
@@ -378,9 +378,6 @@ pubspec.lock
 !.devcontainer/recommended-devcontainer.json
 !.devcontainer/recommended-Dockerfile
 .settings/
-.vscode/launch.json
```

Comment:
Anything in `.vscode/` should be included in the repo for shared project configuration.

Reply:
Good catch!